packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -771,7 +750,7 @@ export class VerticalStackedBarChartBase extends React.Component<
     );
     const shouldFocusWholeStack = this._toFocusWholeStack(_isHavingLines);
 
-    if (this._xAxisType === XAxisTypes.StringAxis && this.props.barWidth === 'auto') {
+    if (this._xAxisType === XAxisTypes.StringAxis) {
       // Setting the bar width here is safe because there are no dependencies earlier in the code
       // that rely on the width of bars in vertical bar charts with string x-axis.
       this._barWidth = getBarWidth(this.props.barWidth, this.props.maxBarWidth, xBarScale.bandwidth());
```

Comment:
no change needed in this call to barwidth

Reply:
For the string x-axis scenario, the existing bar width calculation works fine. The only change is in the `getBarWidth` utility function.